parameters/param2012_20160620.pro

```diff
@@ -0,0 +1,45 @@
+t_launch = 64500.
```

Comment:
I would remove the date from the filename as version control will already know when it was changed. This kind of "version control" does not play nice with git as you'd lose the history of the file every time you updated it. You would need a new file with a new date on it. If you really want to keep some sort of revision date in the file then maintain as a comment inside the file and not in the filename. It might also confuse a user since there are two dates in that filename, one refers to FOXSI and the other to the last revision date (which they don't really need to know).